examples/data-objects/multiview/triangle-view/package.json

```diff
@@ -11,14 +11,22 @@
 	},
 	"license": "MIT",
 	"author": "Microsoft and contributors",
+	"type": "module",
+	"exports": {
+		".": {
+			"import": {
+				"types": "./lib/index.d.ts",
+				"default": "./lib/index.js"
+			}
+		}
```

Comment:
```suggestion
	"exports": {
		".": {
			"types": "./lib/index.d.ts",
			"default": "./lib/index.js"
		}
```

Reply:
@tylerbutler - Do you mind if we simplify the exports in a separate pass?

(Consistency is easier for my lamely written conversion and reporting scripts.)

Reply:
Mentioned elsewhere but for completeness: yes, waiting to clean these up in a pass later is fine.